src/MSAL.PCL/Internal/Http/HttpClientFactory.cs

```diff
@@ -35,7 +35,17 @@ internal class HttpClientFactory
         private static HttpClient _client;
         private static readonly object LockObj = new object();
         public static bool ReturnHttpClientForMocks { set; get; }
+        public const long MaxResponseContentBufferSizeInBytes = 1024*1024;
 
+        private static HttpClient CreateHttpClient()
```

Comment:
@rayluo - your code is functionally incorrect. When mocks are not in play, your suggested code will create a new httpclient every time. But @SomkaPe's implementation keeps the actual intent intact.